src/services/style-service.ts

```diff
@@ -238,6 +238,10 @@ const createStyleService = (theme: ExtendedTheme, layoutService: LayoutService):
     DEFAULT_CELL_HEIGHT,
   );
 
+  styleService.contentRowHeight = styleService.contentCellHeight / lineClamp;
```

Comment:
```suggestion
  styleService.contentRowHeight = (styleService.contentCellHeight - CELL_PADDING_HEIGHT) / lineClamp;
```
shouldn't this give you the correct row height?
and then the `styleService.contentTextHeight` would be the same as `styleService.contentRowHeight` and wont be needed any more?

Reply:
Both are needed in `src/pivot-table/components/cells/DimensionValue/Text.tsx:16`

Reply:
the thing that got me thinking was that do we add padding to **each row** or for **each cell**? 
if it is cell -> then i think we should subtract it like the suggestion, right?

Reply:
Padding is added once to `contentCellHeight`. Even if `lineClamp` is `10`. `contentRowHeight` in a cell is the rows in which text can be draw at. So the calculations should be correct.

Note that a cell that is expanded can have a rendered "cell height" which is `contentCellHeight * <number-of-leaf-nodes>`. But for the context of these values, that does not matter.